### ISSUE TITLE
Allow version-less modules in environments

### ISSUE THREAD

**Issue by fabiangeisler** [cpenv/tk-cpenv#27]
It would be nice to keep some modules always at the "latest" version. For example the Maya scanner plugin which scans Maya scenes for malware content. At the moment I would need to update this module for all environments individually.

## Proposed changes
Adding an extra entry to each module drop down called "latest", which in turn would save the module as version-less in the environment . 

## Notes
Removing the version from a module in an environment requires string, already resolved to the latest version in the GUI. But saving the environment adds the version again.

**Comment by danbradham** [comment]
Yeah this would be a great feature to have. Agree with your proposed change entirely.

**Comment by RicardoMusch** [comment]
Yes would be grand, I modify the env manually to to add modules that I might not have yet published.
UI Support would be good :)
